### PR TITLE
DatastoreServiceImpl create method Unit testing

### PR DESCRIPTION
**Pull request by soumiksgithub**
- This is for VMWare north bound plugin.
- Out of three modules, this UT is for plugin module.
- This is the pull request for Unit testing DatastoreServiceImpl create method.
- This is not having the full coverage of DatastoreServiceImpl.
- Attached the screenshot for coverage.
- I will add class description and method description in Next PR.
image

### CODE REVIEW

File: vmware/ngc/NGC-Plugin/src/test/java/service/impl/DatastoreTest.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/**
+ * 
+ */
+
+package service.impl;
```
Comment (NajmudheenCT) [inline vmware/ngc/NGC-Plugin/src/test/java/service/impl/DatastoreTest.java]: Can we maintain the same package structure as source , like  org.opensds.vmware.ngc.service.impl;

**Comment by soumiksgithub** [comment]
> A small comment on package order.
> Can we also see if these tests can be included in the build script.(can be planned in next PRs also)

Will do it in coming PR. Noted and mailed to you as remonder.

File: vmware/ngc/NGC-Plugin/src/test/java/org/opensds/vmware/ngc/service/impl/DatastoreTest.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
+/**
+ * 
+ */
+
+package service.impl;
```
Comment (NajmudheenCT) [inline vmware/ngc/NGC-Plugin/src/test/java/org/opensds/vmware/ngc/service/impl/DatastoreTest.java]: Please change to "package org.opensds.vmware.ngc.service.impl;"

Comment (soumiksgithub) [reply vmware/ngc/NGC-Plugin/src/test/java/org/opensds/vmware/ngc/service/impl/DatastoreTest.java]: Was my bad. Changed now. please check

**Pull request merged**